### ISSUE TITLE
"unused-dependency" rule doesn't work version catalogs

### ISSUE THREAD

**Issue by zman0900** [nebula-plugins/gradle-lint-plugin#390]
When using Gradle's "version catalog" feature to declare dependencies, the "unused-dependency" detector doesn't find problems that it normally would.  

With `gradle/libs.versions.toml` file containing:
```toml
[versions]
slf4j = "1.7.36"

[libraries]
slf4j-api = { group = "org.slf4j", name = "slf4j-api", version.ref = "slf4j" }
```

Nothing is detected when dependency is declared  like this on a project that does not use slf4j-api:
```gradle
dependencies {
    implementation libs.slf4j.api
}
```

But if dependency is declared like this instead, it is correctly detected as unused:
```gradle
dependencies {
    implementation 'org.slf4j:slf4j-api:1.7.36'
}
```

**Comment by sembozdemir** [comment]
I have the same issue. is there any plan to support version-catalogs?
